sr_robot_launch/scripts/construct_robot_description

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/python
+
+# Copyright 2019 Shadow Robot Company Ltd.
+#
+# This program is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by the Free
+# Software Foundation version 2 of the License.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program. If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import rospy
+import rospkg
+from subprocess import check_output
+from sr_ur_arm_calibration_loader.sr_ur_arm_calibration_loader import SrUrLoadCalibration
+
+
+class SrConstructRobotDescription():
+    def __init__(self):
+        self._robot_description_file = rospy.get_param("~robot_description_file")
+        CONST_ARM_TYPE = rospy.get_param("~arm_type")
+        self._side = ''
+        self._robot_description_params = {}
+        self._node_name = rospy.get_name()
+        self._arm_ra = {'prefix': 'ra', 'ip_address': '192.168.1.1', 'arm_type': CONST_ARM_TYPE}
```

Comment:
I'd move this to be a local var (only used in one method) and make const_arm_type member, that way we reduce number of unnecessary member variables

take a look at:
https://softwareengineering.stackexchange.com/questions/388052/rationale-to-prefer-local-variables-over-instance-variables

"The smaller the scope, the better. The rationale is that variables should live in the smallest scope possible. There are many benefits to this:"